spring-boot-starter/external-task-client/loan-granting-spring-boot-webapp/pom.xml

```
@@ -18,7 +18,7 @@
     <maven.compiler.plugin.version>3.11.0</maven.compiler.plugin.version>
 
     <graaljs.version>21.1.0</graaljs.version>
-    <jaxb-api.version>2.3.1</jaxb-api.version>
+    <jaxb.version>4.0.4</jaxb.version>
```

🔧 Similar point to the above
```suggestion
    <jaxb.version>4.0.5</jaxb.version>
```